src/main/java/com/treasure/hunt/strategy/hider/impl/HintEvaluator.java

```diff
@@ -0,0 +1,142 @@
+package com.treasure.hunt.strategy.hider.impl;
+
+import com.treasure.hunt.strategy.hint.impl.AngleHint;
+import javafx.util.Pair;
+import lombok.extern.slf4j.Slf4j;
+import org.locationtech.jts.algorithm.Angle;
+import org.locationtech.jts.geom.Coordinate;
+
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Slf4j
+public class HintEvaluator {
+
+    private List<AngleHintStat> hints;
+    private List<Pair<Coordinate, Pair<Double, Double>>> treasurePointsOfInterest; //Coordinate, Constant, Angle under PlayerPosition
+    private Coordinate playerPosition;
+    private Coordinate origin = new Coordinate(0,0);
+    private double currentArea;
+
+    //algorithmic parameters
```

Comment:
better use java docs

Reply:
Is it good style to JavaDoc private variables?